readme.html

```diff
@@ -136,141 +136,210 @@
 	</ul>
 </p>
 
-<p><b>List of indicator types (itype) for the import_&lt;indicator_type&gt;_observable and the import_observables actions (To be given as input when with_approval parameter is set to False)</b>
```

Comment:
This change here is simply UI update from bullet point display to multi column display:
![Screen Shot 2021-12-03 at 4 27 24 PM](https://user-images.githubusercontent.com/82124762/144701858-5069f88f-98d6-4173-a3f9-2a932847d049.png)